src/main/resources/application.yml

```diff
@@ -8,3 +8,10 @@ spring:
       mail.smtp.auth: true
       mail.smtp.starttls.enable: true
       mail.smtp.ssl.trust: smtp.gmail.com
+
+  mvc:
+    throw-exception-if-no-handler-found=true:
+server:
+  error:
+    whitelabel:
+      enabled=false:
```

Comment:
Pytanko, co to jest i jak działa? :)

Reply:
to blokuje takie automatyczne białe strony z errorami -> wtedy mogą dojść do głosu te twoje.